.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -488,6 +488,7 @@ private double GetIntValue(string intStr)
             var isRoundBefore = false;
             long roundBefore = -1, roundDefault = 1;
             var isNegative = false;
+            var isPreDigit = false;
```

Comment:
Can I name it  as "isDigitBefore"?

Reply:
isMultiDigit means that it's not a single digit, then it must have previous digits.
Or it could be hasPreviousDigits too.